src/task1.java

```diff
@@ -0,0 +1,209 @@
+package com.example.helloworld;
+import java.util.*;
+
+import com.sun.deploy.security.SelectableSecurityManager;
+import org.apache.commons.lang3.ArrayUtils;
+
+public class Task_1 {
+
+    public static void main(String[] args) {
+        System.out.println("Hello, World!");
+
+        int[] numbers = {1, 5, 10, 24, 55, -20};
+        String[] textArray = {null, "hello", "my", null, "name", "is", "Orest", "hello"};
+
+        getMinMaxValue(numbers);
+        reverseArray(numbers);
+        getDuplicateValuesInArray(textArray);
+        removeDuplicateValuesFromArray(textArray);
+        getNameOfWeekDay();
+        getNumberOfDaysInMonth();
+    }
+
+    // 1. Write a Java program to find the maximum and minimum value of an array
+    public static void getMinMaxValue(int[] numbers) {
+
+        if (numbers == null || numbers.length < 1) {
+            System.out.println("method getMinMaxValue parameter 'numbers' is: null or 0 length");
+            return;
+        }
+
+        int maxValue = numbers[0], minValue = numbers[0];
+
+        for (int i = 1; i < numbers.length; i++) {
+            if (numbers[i] > maxValue) {
+                maxValue = numbers[i];
+            }
+            if (numbers[i] < minValue) {
+                minValue = numbers[i];
+            }
+        }
+        System.out.println("Minimum value is : " + minValue + " and Maximum value is : " + maxValue);
+    }
+
+    // 2. Write a Java program to reverse an array of integer values
+    public static void reverseArray(int[] numbers) {
+
+              // first wanted like this but then found that there is such Apache Commons Lang
+              //  int[] b = new int[numbers.length];
+              //  int j = numbers.length;
+              //  for (int i = 0; i < numbers.length; i++) {
+              //      b[j - 1] = numbers[i];
+              //      j = j - 1;
+              //  }
+
+              //  System.out.println("Reversed array - " + Arrays.toString(b));
+
+        ArrayUtils.reverse(numbers);
+        System.out.println("Reversed array - " + Arrays.toString(numbers));
+    }
+
+    // 3. Write a Java program to find the duplicate values of an array of string values
+    public static void getDuplicateValuesInArray(String[] textArray) {
+
+        for (int i = 0; i < textArray.length-1; i++) {
+            for (int j = i + 1; j < textArray.length; j++) {
+                try {
+                    if( (textArray[i].equals(textArray[j])) && (i != j) ) {
+                        System.out.println("Duplicate Element is : " + textArray[j]);
+                    }
+                } catch (Exception e) {
```

Comment:
Suppose you need to add some validation because you know that there will be an incorrect element with `null`